docs/fast_data/tutorials/low-code.mdx

```diff
@@ -484,51 +465,102 @@ It is important to also define the JSON Schema for the _order_ field of the Sing
 }
 ```
 
-:::info
-We can of course create more fields: here we omitted the email, phone and address fields because we don't need to expose them in our Single View.
-:::
-![Single View Fields](img/21-single-view-fields.png)
+Then press on the **Save** button on the bottom right of the page to apply the changes made to the field.
 
-#### Strategies
+</p>
+</details>
 
-Now we need to create the strategies of the Single View. Strategies are used to define when the data need to be updated in a Single View. For example, when we update the registry we need to update the Single View information.
-In this tutorial, we will create two strategies: the first one is for the registry's update and the second one is for the order's update.
+### Microservice Configuration
 
-Inside our Single View "order-history", click on the **Add link to strategy** button.
-![Strategy Page](img/28-strategy-page.png)
+First of all, we need to create a microservice that we will attach to the Single View that we're going to create. This microservice is called "Single View Creator", and it's available in the microservice Marketplace.
 
-Now we will choose the System of Record and the Projection that we want to link to the Strategy; in our case, "delivery" and "registry".
-![Strategy Page](img/29-strategy-select-projection.png)
+![Marketplace and Single View Creator](img/23-marketplace-single-view-low-code.png)
 
-Last but not least, we need to choose the Strategy Source. In this case, we will use the "Low Code" strategy, then we can click on the **Create** button.
-![Strategy Page](img/30-strategy-low-code.png)
+Here we have two different plugins: the first one is for the Single View Creator, and the second one is for the Single View Creator Low Code.
+We will go for the latter since we want to leverage the usage of ConfigMaps for a quicker configuration of the microservice. We suggest naming it `single-view-creator-delivery`.
```

Comment:
Maybe you refer to no code configuration canvas?

Reply:
I added both low-code and no-code with the corresponding doc page